Sascha-L/WPF-MediaKit
# Add support for .NET 6

Motivation
----------
Allow Windows consumers to use .NET 6 or later instead of .NET 4.

Modifications
-------------
- Move to the modern .csproj format with multi-targeting
- Drop the .nuspec file and instead add properties directly to the csproj
- Update to latest C++ platform toolset
- Remove the unused settings files
- Fix some miscellaneous warnings and compatibility issues
- Set Visible to False for the DLLs in the targets file so they don't appear in Visual Studio and only get copied to output
- Use `buildTransitive` so that when WPFMediaKit is a transitive dependency the required DLLs are still copied
- Add SourceLink support and produce snupkg symbol packages
- Embed the license and readme in the package
- Bump version to 2.3.0

Results
-------
The NuGet package can now be produced by the .NET SDK 6 or greater with:

```
dotnet pack -c Release .\Source
```

The resulting package includes .NET 4.6.2 targets (oldest framework still in support by Microsoft) and .NET 6 targets.
